docs/adapters.md

```diff
@@ -1,4 +1,7 @@
-# Hubot Adapters
+---
+permalink: /docs/adapters/index.html
+layout: docs
```

Comment:
I tried that in hubot.github.com's _config.yml, but it didn't seem to take effect:

```yaml

defaults:
  -
    scope:
      path: "docs"
      type: "submodules"
    values:
      layout: "docs"
```